packages/button/addon/templates/components/nucleus-button.hbs

```diff
@@ -1,4 +1,8 @@
-{{#if icon}}{{nucleus-icon name=icon size=_iconSize}}{{/if}}
+{{#if icon}}
+  <span class="nucleus-button__icon {{unless iconOnly "active"}}">
```

Comment:
A class named `active` is quite common and the consuming apps might override our styles. So can we make this class a little more nucleus specific?